tests/e2e_logging.py

```diff
@@ -15,12 +15,18 @@
 @reqs.at_least_n_nodes(2)
 def test(network, args, notifications_queue=None, verify=True):
     txs = app.LoggingTxs(notifications_queue=notifications_queue)
-    txs.issue(network=network, number_txs=1, wait_for_sync=args.consensus == "raft")
+    txs.issue(
+        network=network,
+        number_txs=1,
+        wait_for_sync=args.consensus == "raft",
+        consensus=args.consensus,
```

Comment:
There's already a `wait_for_sync` argument that depends on the consensus. Can you collapse the two arguments?

Reply:
We could collapse the two arguments, but will there be a case where we want to set wait for sync independently from the consensus being used?

Reply:
The `wait_for_sync` flag was added as a quick fix for PBFT so we can remove it safely.

Reply:
awesome